addons/kommander/1.4/kommander.yaml

```
@@ -9,7 +9,7 @@ metadata:
     # This was originally added to support the PVC's needed for the Kubecost subcomponent.
     kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.4.2-3"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.4.2-4"
     appversion.kubeaddons.mesosphere.io/kommander: "1.4.2"
```

```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "1.4.3-1"
    appversion.kubeaddons.mesosphere.io/kommander: "1.4.3"
```

I think we're on 1.4.3 now